coderd/prebuilds/preset_snapshot.go

```diff
@@ -31,9 +31,14 @@ const (
 // PresetSnapshot is a filtered view of GlobalSnapshot focused on a single preset.
 // It contains the raw data needed to calculate the current state of a preset's prebuilds,
 // including running prebuilds, in-progress builds, and backoff information.
+// - Running: prebuilds running and non-expired
+// - Expired: prebuilds running and expired due to the preset's TTL
+// - InProgress: prebuilds currently in progress
+// - Backoff: holds failure info to decide if prebuild creation should be backed off
 type PresetSnapshot struct {
 	Preset        database.GetTemplatePresetsWithPrebuildsRow
 	Running       []database.GetRunningPrebuiltWorkspacesRow
```

Comment:
I’m not a fan of the `Running` field name here, it’s a bit misleading since now it only includes non-expired running prebuilds. Maybe something like `RunningValid` would make that clearer. Open to other naming ideas too if anyone has suggestions. 
I can make this change in a follow-up PR to avoid adding more entropy here.